spinalcordtoolbox/reports/assets/index.html

```diff
@@ -241,9 +246,37 @@
          for the case where the table should be empty (i.e. when search
          results correspond to nothing).
          */
+        b=document.createElement("div")
+        b.className = "fixed-table-toolbar";
+        b.style.paddingTop="10px";
+
+        btn1 = document.createElement("button");
+        btn1.type="button";
+        btn1.classList ="btn btn-default";
+        btn1.id="download_yaml_btn_274c";
+        btn1.onclick = function() { download_yaml('274c') }; 
+        btn1.disabled="disabled";
+        btn1.innerText="Download QC Fails";
+
+        btn2 = document.createElement("button");
+        btn2.type="button";
+        btn2.classList ="btn btn-default";
+        btn2.id="download_yaml_btn_26a0";
+        btn2.onclick = function() { download_yaml('26a0') }; 
+        btn2.disabled="disabled";
+        btn2.innerText="Download QC Artifacts";
+        b.appendChild(btn1);
+        b.appendChild(btn2);
+        
+        a = document.getElementsByClassName("bootstrap-table");
+        a[0].appendChild(b);
+        c = a[0].children[2];
+        a[0].children[2] = b;
+        a[0].children[3] = c;
```

Comment:
This section of code fails for me:

![Screenshot from 2021-04-20 09-33-50](https://user-images.githubusercontent.com/16181459/115404671-8c50e480-a1bb-11eb-9378-48f832e8390f.png)

I set a breakpoint and took a look at `a`, and it's returning an empty collection:

![Screenshot from 2021-04-20 09-35-25](https://user-images.githubusercontent.com/16181459/115404957-cb7f3580-a1bb-11eb-9004-3b9837d77ad9.png)

So, the div element `b` never gets added, hence explaining the other failure.

Reply:
Elsewhere, we identify the table using `"table"` rather than `"bootstrap-table"`:

https://github.com/neuropoly/spinalcordtoolbox/blob/55d7d78b6357d6c2b61b6ab991db2a54aae08f7c/spinalcordtoolbox/reports/assets/_assets/js/main.js#L73-L74

So, I'm going to try using that instead and see if it fixes things.

Reply:
What is strange is that you didn't have that issue before we were making all these changes in this PR. @TommyBoshkovski do you have any clue?

Reply:
Yeah, there is always an issue with Mozilla since they don't always have the same html/css attributes. I will look into the issue and will try to make it Mozilla compatible.

Reply:
@TommyBoshkovski I've been looking into this (sorry!) but could you please check https://github.com/neuropoly/spinalcordtoolbox/pull/3253#discussion_r616737845?